doc/whats_new/v1.4.rst

```diff
@@ -35,6 +35,13 @@ random sampling procedures.
       solvers (when fit on the same data again). The amount of change depends on the
       specified `tol`, for small values you will get more precise results.
 
+- |Enhancement| :class:`decomposition.PCA`, :class:`decomposition.SparsePCA` and
```

Comment:
I am wondering if it has an impact on what was reported in https://github.com/scikit-learn/scikit-learn/issues/27620 when using MKL.